common/include/pcl/point_types_conversion.h

```diff
@@ -369,6 +369,25 @@ namespace pcl
     * \param[in] in the input XYZRGB point cloud
     * \param[out] out the output XYZHSV point cloud
     */
+  
+  inline void 
+  PointCloudXYZHSVtoXYZRGB (const PointCloud<PointXYZHSV>& in,
+                            PointCloud<PointXYZRGB>&       out)
+  {
+    out.width   = in.width;
+    out.height  = in.height;
+    for (const auto &point : in.points)
+    {
+      PointXYZRGB p;
+      PointXYZHSVtoXYZRGB (point, p);
+      out.push_back (p);
+    }
+  }
+
+  /** \brief Convert a XYZHSV point cloud to a XYZRGB
```

Comment:
It seems you have mixed up the documentation. Please keep the doc for the existing function the same, and add the appropriate doc for the new function.
Side note: don't worry about the failing test, that unfortunately happens sometimes (rarely). We didn't find a good solution for that yet. If you push the fix for this, the check will run again and most likely pass

Reply:
Hi,

Check my last commit and please tell me if everything is now correct.

Sorry for the inconvenience.
G